frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HypershiftUpgradeModal.tsx

```diff
@@ -68,7 +68,7 @@ export function HypershiftUpgradeModal(props: {
   const hypershiftSupportedVersionsConfigMap = configMaps.find(
     (cm) => cm.metadata?.name === 'supported-versions' && cm.metadata?.namespace === 'hypershift'
   )
-  const supportedVersions = JSON.parse(_.get(hypershiftSupportedVersionsConfigMap, 'data.supported-versions', '{}'))
+  const supportedVersions = JSON.parse(_.get(hypershiftSupportedVersionsConfigMap, 'data.supported-versions', '[]'))
```

Comment:
This doesn't look correct. `supportedVersions.versions` is an array, but `supportedVersions` is not.